Cloudfront-event-engine-assets/serverless-saas-userinterface.yaml

```diff
@@ -0,0 +1,239 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
As discussed, create a folder for event-engine-assets and move this file along with pre-requisites-event-engine.sh and prereq-sam-template.yaml into it. I think we can also move Lab1/event-engine-assets/packaged.yaml into this folder. Just make sure to give proper name while moving.

Reply:
Done.